### DISCUSSION TITLE
紧急求助：115网盘提示“请升级到最新版本: unexpected error”

### DISCUSSION THREAD

**Discussion by zhsxh** [AlistGo/alist#7211] [Q&A] [answered]
115浏览器我已经更新到最新版本，alist我也更新到最新的v3.37.3版本，但是，从夸克、阿里云等往115复制文件时，仅能复制一级目录，其他任何文件都复制不过去，提示“请升级到最新版本: unexpected error”。但是夸克、阿里云之间复制文件没有影响。求教大佬，这是怎么回事，如何解决啊？在线等大佬赐教。

![微信图片_20240918122832](https://github.com/user-attachments/assets/2229fe94-2405-4475-a3c2-bec553c34d2c)

![微信图片_20240918122803](https://github.com/user-attachments/assets/a14f72f7-bd8f-4648-828b-a91c162190fa)

![微信图片_20240918122836](https://github.com/user-attachments/assets/18565685-7151-4c08-8f6d-c8fdef524c8d)

**Comment by zhsxh** [comment]
而且，登陆alist账号时提示“Token is invalidated”。目前只有115不好用，图片、视频等各种文件都打不开，夸克、阿里云等网盘没有问题。

**Reply by 1305854203** [reply]
我也是更新了v3.37.3，复制文件失败，真的急死了

**Reply by sun3u** [reply]
+1

**Reply by Sxiaoai** [reply]
+1

**Comment by anwen-anyi** [comment, marked as answer]
更新 v3.37.4
![image](https://github.com/user-attachments/assets/cec97734-56f7-4921-a2d2-6daa824aefc6)
![image](https://github.com/user-attachments/assets/1baf6d88-8af0-44df-a4ec-b9e34eade24a)

**Reply by zhsxh** [reply]
确实，更新到 v3.37.4以后好用了。非常感谢大神。
